scripts/flake_checker_test.py

```diff
@@ -60,35 +102,40 @@ def mock_getenv(unused_variable):
 
 class MockDatetime:
 
-    def __init__(self, date):
+    def __init__(self, date: datetime.datetime) -> None:
         self.date = date
 
-    def utcnow(self):
+    def utcnow(self) -> datetime.datetime:
         """Get datetime.datetime object."""
         return self.date
 
 
 class ReportPassTests(test_utils.GenericTestBase):
 
-    def setUp(self):
+    def setUp(self) -> None:
         super(ReportPassTests, self).setUp()
         self.example_date = datetime.datetime(2020, 1, 1, 0, 0, 0, 1)
 
-    def test_successful_report(self):
+    def test_successful_report(self) -> None:
 
-        def mock_getenv(variable):
-            environment_vars = {
+        def mock_getenv(variable: str) -> Union[str, int, None]:
+            environment_vars: Dict[str, Union[str, int]] = {
                 'CIRCLECI': 1,
                 'CIRCLE_USERNAME': 'user',
                 'CIRCLE_BUILD_URL': 'https://example.com',
                 'CIRCLE_BRANCH': 'develop',
             }
             return environment_vars.get(variable)
 
-        def mock_post(url, json, allow_redirects, headers):  # pylint: disable=unused-argument
+        def mock_post(
+            url: str,  # pylint: disable=unused-argument
```

Comment:
I think we can take away the `pylint` ignore by either replacing the unused argument with `_` or an argument name that starts with `_`. 

For eg: `_url`

Ditto elsewhere.

Reply:
done